index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+
+<body>
+    <header class="header">
+        <div class="logo-container">
+            <a href="#" class="my-logo">My Logo</a>
+        </div>
+        <nav class="menu">
+            <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
+        </nav>
+    </header>
+    <main class="main-container">
+        <section class="headline">
+            <h1 class="headline-title">I'm Misikir<br> Glad to see you!</h1>
+            <p class="headline-description">I’m a software developer! I can help you build a product , feature or
+                website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <h2 class="connect-link">LET’S CONNECT</h2>
```

Comment:
- [ ] If you notice `Let's connect` is a link that should direct you to contact area like social icons or contact form :smirk: . This implies that the text should be a link and hence `a` tag should be used instead of `<h2>` tag. For [Semantic purposes](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwi56cjW38D8AhX0UaQEHVp7DLEQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU), kindly ensure that the text is a link to any of any of suggestions( social media or any contact element).